### PR TITLE
add DigitsWriter to data reader

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: Detectors/FIT/FT0/workflow/src/FT0DigitWriterSpec.cxx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using namespace o2::framework;
+
+namespace o2
+{
+namespace ft0
+{
+
+template <typename T>
+using BranchDefinition = MakeRootTreeWriterSpec::BranchDefinition<T>;
+DataProcessorSpec getFT0DigitWriterSpec()
```
Comment (sawenzel) [inline Detectors/FIT/FT0/workflow/src/FT0DigitWriterSpec.cxx:28]: There is an existing FT0DigitWriterSpec in `Steer/DigitizerWorkflow`.
I agree that we should move detector specific components away from there. So could you make the transition proper and migrate completely?

**Review by sawenzel — CHANGES_REQUESTED** [review]

File: Detectors/FIT/FT0/workflow/src/FT0DigitWriterSpec.cxx (hunk 1 above)
Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/workflow/src/FT0DigitWriterSpec.cxx:28]: Hello
I have 2 points:
1. I can not open  log of clang errors
2. Its not really transition, just conversion raw data to digits  for simplifying next QC and calibration.

**Comment by AllaMaevskaya** [comment]
I think build/o2checkcode/o2 error is not related to our code

**Comment by sawenzel** [comment]
@AllaMaevskaya : I am still not completely happy that you will now have 2x FT0DigitWriterSpec in the code base  or maybe it's just an unfortunate naming and in reality these 2 things do different stuff. I'll merge it now, but maybe you could target a code unification?

**Comment by AllaMaevskaya** [comment]
OK, we will think how to use one DigitsWriter.
Thank you
Alla

**Pull request merged**